recipes/android-ndk/config.yml

```diff
@@ -1,6 +1,9 @@
----
-    versions:
-      "r21d":
-        folder: all
-      "r22":
-        folder: all
+versions:
```

Comment:
I am not sure if it is possible to add more than 1 additional version into 1 PR

Reply:
it is possible